lib/holdingsiq.js

```diff
@@ -0,0 +1,345 @@
+  data.forEach((e) => {
+    const result = {
+      ezhlmid: `${institute}-${e?.customerResourcesList?.[0]?.packageId}-${e?.customerResourcesList?.[0]?.vendorId}-${e?.titleId}`,
+      PackageID: e?.customerResourcesList?.[0]?.packageId,
+      VendorID: e?.customerResourcesList?.[0]?.vendorId,
+      KBID: e?.titleId,
+      PackageName: e?.customerResourcesList?.[0]?.packageName,
+      ManagedCoverageBegin: e?.customerResourcesList?.[0]?.managedCoverageList?.beginCoverage,
+      ManagedCoverageEnd: e?.customerResourcesList?.[0]?.managedCoverageList?.endCoverage,
+      CustomCoverageBegin: e?.customerResourcesList?.[0]?.customCoverageList?.beginCoverage,
+      CustomCoverageEnd: e?.customerResourcesList?.[0]?.customCoverageList?.endCoverage,
+      Publisher: e?.publisherName,
+      Embargo: getEmbargo(e?.customerResourcesList?.[0]?.managedEmbargoPeriod),
+      CustomEmbargo: getEmbargo(e?.customerResourcesList?.[0]?.customEmbargoPeriod),
+      VendorName: e?.customerResourcesList?.[0]?.vendorName,
+      OnlineISBN: getIdentifier(e?.identifierList, 1, 2),
+      PrintISBN: getIdentifier(e?.identifierList, 1, 1),
+      OnlineISSN: getIdentifier(e?.identifierList, 0, 2),
+      PrintISSN: getIdentifier(e?.identifierList, 0, 1),
+      HideOnPublicationFinder: e?.customerResourcesList?.[0]?.visibilityDate?.isHidden,
+      Title: e?.titleName,
+      PackageType: e?.packageType,
+      ResourceType: e?.pubType,
+      URL: e?.customerResourcesList?.[0]?.url,
+      Author: getContributor(e?.contributorsList, 'Author'),
+      PeerReviewed: e?.isPeerReviewed,
+      Subject: e?.subjectList?.subjects,
+      IsCustom: e?.isTitleCustom,
+      UserDefinedField1: e?.customerResourcesList?.[0]?.UserDefinedField1,
+      UserDefinedField2: e?.customerResourcesList?.[0]?.UserDefinedField2,
+      UserDefinedField3: e?.customerResourcesList?.[0]?.UserDefinedField3,
+      UserDefinedField4: e?.customerResourcesList?.[0]?.UserDefinedField4,
+      UserDefinedField5: e?.customerResourcesList?.[0]?.UserDefinedField5,
+      IsProxyInherited: e?.customerResourcesList?.[0]?.proxy?.inherited,
+      HiddenBy: e?.customerResourcesList?.[0]?.visibility?.reason,
+      SelectedBy: e?.customerResourcesList?.[0]?.selectedBy,
+      CustomCoverageBy: e?.customerResourcesList?.[0]?.customCovergeOwnership,
+      CoverageStatement: e?.customerResourcesList?.[0]?.CoverageStatement,
+      Description: e?.description,
+      Editor: getContributor(e?.contributorsList, 'Editor'),
+      Edition: e?.edition,
+      Illustrator: getContributor(e?.contributorsList, 'Illustrator'),
+      AlternateTitle: e?.alternateTitle,
+      CoveragePackage: e?.customerResourcesList?.[0]?.isPackageCustom,
+      Proxy: e?.customerResourcesList?.[0]?.isPackageCustom,
+      TitleSelected: e?.customerResourcesList?.[0]?.isSelected,
+    };
+
+    results.push({ update: { _index: index, _id: result.ezhlmid } });
+    results.push({ doc: result });
+  });
+  return results.slice();
```

Comment:
```suggestion
  return results;
```

No need to slice the array here